store/migration/dev/20220715000000__primary_key.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE idx ADD COLUMN "primary" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
The schema change should be a separate PR by following https://github.com/bytebase/bytebase/blob/main/docs/schema-update-guide.md#development

Reply:
This PR only contains the four steps in https://github.com/bytebase/bytebase/blob/main/docs/schema-update-guide.md#development.

So, how should I understand ‘separate’

Reply:
The sql files under store directory can be a separate PR. We can focus on schema review (LOL) in that PR. This PR can focus on code review.

Reply:
Got it. We'd better add this information to https://github.com/bytebase/bytebase/blob/main/docs/schema-update-guide.md#development.

Reply:
I'll modify this PR now.